### PR TITLE
Update rooms-licensing.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6528
Added link

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-licensing.md [hunk 1]
```diff
@@ -69,4 +69,6 @@ The following table lists the features that are available in Microsoft Teams Roo
 > [!NOTE]
 > If you have existing licenses assigned for room systems, these will continue to work without any interruption. You should move to use the new Meeting Room SKU when existing licenses expire.  
 
- **Use the right version of Windows 10**: For customers who want to deploy Windows 10 images to their devices, see [Configure a Microsoft Teams Rooms console](https://docs.microsoft.com/microsoftteams/room-systems/console). You can get a copy from the [Volume Licensing Service Center](https://www.microsoft.com/Licensing/servicecenter/).
+ **Use the right version of Windows 10**: For customers who want to deploy Windows 10 images to their devices, see [Configure a Microsoft Teams Rooms console](https://docs.microsoft.com/microsoftteams/room-systems/console). You can get a copy from the [Volume Licensing Service Center](https://www.microsoft.com/Licensing/servicecenter/). 
+ 
+ See also [Great meeting room experiences: Meet the new Microsoft Teams Rooms Standard and Premium](https://www.microsoft.com/microsoft-365/blog/2020/07/21/microsoft-teams-meetings-hybrid-workplace-options/).
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-licensing.md:74]:
```suggestion
 See also [Great meeting room experiences: Meet the new Microsoft Teams Rooms Standard and Premium](https://www.microsoft.com/en-us/microsoft-365/blog/2020/07/21/microsoft-teams-meetings-hybrid-workplace-options/).
```

Comment (JohanFreelancer9) [reply Teams/rooms/rooms-licensing.md:74]: Hi Lindsay - for blogs, the local needs to be included in the link. If not, the link will not work in some countries.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**